pkg/server/createExperiment.go

```diff
@@ -143,3 +151,42 @@ func ProcessCreateQueue(q chan Request) {
 		go processRequest(&req)
 	}
 }
+
+func SaveRecommendationData(containerMapObject map[string][]string, req *Request) *RecommendationData {
+
+	var recommendationResult = RecommendationData{ClusterNamespace: make(map[string]string),
+		RecommendationID: make(map[string]string), RecommendationStatus: make(map[string]string),
+		Recommendation: make(map[string][]map[string][]map[string]string)}
+
+	reqParts := strings.Split(req.RequestName, "_")
+	recid := reqParts[3]
+
+	clusterNamespace := fmt.Sprintf("%s_%s", reqParts[1], reqParts[2])
+
+	recommendationResult.ClusterNamespace[clusterNamespace] = recid
+	recommendationResult.RecommendationID[recid] = clusterNamespace
+
+	deployments := make([]map[string][]map[string]string, 0)
+
+	for deployment, containers := range containerMapObject {
+		containerRecommendations := make([]map[string]string, 0)
+		for _, container := range containers {
+			containerRecommendation := map[string]string{
+				container: "recommendation-status",
+			}
+			containerRecommendations = append(containerRecommendations, containerRecommendation)
+		}
+
+		deploymentMap := map[string][]map[string]string{
+			deployment: containerRecommendations,
+		}
+		deployments = append(deployments, deploymentMap)
+	}
+
+	recommendationResult.Recommendation[recid] = deployments
+
+	klog.Info(recommendationResult)
```

Comment:
Please add a message to provide context in this log.
```suggestion
	klog.Info("Added recommendation. ", recommendationResult)
```

Reply:
Considering using higher verbosity because it may log too much data.